tests/backup/backup_helper.go

```diff
@@ -1443,3 +1452,207 @@ func DeleteBackupAndWait(backupName string, ctx context.Context) error {
 	_, err := task.DoRetryWithTimeout(backupDeletionSuccessCheck, backupDeleteTimeout, backupDeleteRetryTime)
 	return err
 }
+
+// GetPxBackupVersion return the version of Px Backup as a VersionInfo struct
+func GetPxBackupVersion() (*api.VersionInfo, error) {
+	ctx, err := backup.GetAdminCtxFromSecret()
+	if err != nil {
+		return nil, err
+	}
+	versionResponse, err := Inst().Backup.GetPxBackupVersion(ctx, &api.VersionGetRequest{})
+	if err != nil {
+		return nil, err
+	}
+	backupVersion := versionResponse.GetVersion()
+	return backupVersion, nil
+}
+
+// GetPxBackupVersionString returns the version of Px Backup like 2.4.0-e85b680
+func GetPxBackupVersionString() (string, error) {
+	backupVersion, err := GetPxBackupVersion()
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%s.%s.%s-%s", backupVersion.GetMajor(), backupVersion.GetMinor(), backupVersion.GetPatch(), backupVersion.GetGitCommit()), nil
+}
+
+// GetPxBackupVersionSemVer returns the version of Px Backup in semver format like 2.4.0
+func GetPxBackupVersionSemVer() (string, error) {
+	backupVersion, err := GetPxBackupVersion()
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%s.%s.%s", backupVersion.GetMajor(), backupVersion.GetMinor(), backupVersion.GetPatch()), nil
+}
+
+// GetPxBackupBuildDate returns the Px Backup build date
+func GetPxBackupBuildDate() (string, error) {
+	ctx, err := backup.GetAdminCtxFromSecret()
+	if err != nil {
+		return "", err
+	}
+	versionResponse, err := Inst().Backup.GetPxBackupVersion(ctx, &api.VersionGetRequest{})
+	if err != nil {
+		return "", err
+	}
+	backupVersion := versionResponse.GetVersion()
+	return backupVersion.GetBuildDate(), nil
+}
```

Comment:
You can use `GetPxBackupVersion` directly and then do `backupVersion.GetBuildDate`, like in other functions `GetPxBackupVersion***`.